### DISCUSSION TITLE
https://notes.alexkehayias.com/running-docker-compose-in-codespaces/

### DISCUSSION THREAD

**Discussion by giscus** [alexkehayias/note-comments#11] [Announcements]
# https://notes.alexkehayias.com/running-docker-compose-in-codespaces/

Using the built-in docker-compose configuration in GitHub Codespaces is limited. It’s better to run docker-compose inside the Codespace but this requires a docker-in-docker setup which is finicky.

https://notes.alexkehayias.com/running-docker-compose-in-codespaces/

**Comment by ccamarillo** [comment]
There's a missing comma in your `devcontainer.json` sample code.  Thank you for your help.

**Reply by alexkehayias** [reply]
Thank you--updated!
